src/components/ui/CustomCard/index.tsx

```diff
@@ -26,11 +26,21 @@ function CardHeader(props) {
   );
 }
 
+// linkgenerator strips the markdown style links
+// and returns a string with fully formed anchor tag.
+// Use dangerouslySetInnerHTML to inject string as HTML
+function linkgenerator(text: string): string {
+  const regex = /\[([^\]]+)\]\(([^)]+)\)/g;
+  const subst = `<a href="$2" target="_blank">$1</a>`;
+  return text.replace(regex, subst);
+}
+
 function CardBody(props) {
   const { text } = props;
+  const fileteredText = linkgenerator(text);
   return (
     <div className="mx-2 my-6 overflow-y-auto lg:my-8">
-      <p className="text-gray-700 dark:text-gray-100">{text}</p>
+      <p className="text-gray-700 dark:text-gray-100" dangerouslySetInnerHTML={{__html: fileteredText}}></p>
```

Comment:
I think?
```suggestion
      <p className="text-gray-700 dark:text-gray-100" dangerouslySetInnerHTML={{__html: filteredText}}></p>
```

Reply:
Is there a possibility of an injection attack via this?  Is there any way to just search and replace for any md style links that didn't get translated and fix them "by hand" instead?

Reply:
> Is there a possibility of an injection attack via this? Is there any way to just search and replace for any md style links that didn't get translated and fix them "by hand" instead?

@TomSweeneyRedHat this can be done by hand as this is just a one-off substitution.

Reply:
figured that, all could be handled using the pre-existsing `MarkDown` component